packages/flutter/lib/src/cupertino/localizations.dart

```diff
@@ -457,6 +461,9 @@ class DefaultCupertinoLocalizations implements CupertinoLocalizations {
   @override
   String get modalBarrierDismissLabel => 'Dismiss';
 
+  @override
+  String get contextMenuDismissLabel => 'Dismiss context menu';
```

Comment:
I'm thinking from a regular user's perspective maybe "Dismiss menu" would fit better than "Dismiss context menu"? Not sure how many people would call it a "context menu" rather than just "menu" - and how translatable "context menu" is.

Reply:
Alright, that works for me.